go.mod

```diff
@@ -129,4 +129,6 @@ require (
 
 replace github.com/ethereum/go-ethereum => github.com/prysmaticlabs/bazel-go-ethereum v0.0.0-20200626171358-a933315235ec
 
+replace github.com/libp2p/go-libp2p-pubsub => github.com/nisdas/go-libp2p-pubsub v0.3.3-0.20200906105057-0a9a3f646d54
```

Comment:
https://github.com/libp2p/go-libp2p-pubsub/pull/376 landed, so we can probably use vanilla go-libp2p-pubsub package, now?